lib/item.rb

```diff
@@ -1,21 +1,26 @@
 # frozen_string_literal: true
 
+require 'bigdecimal'
```

Comment:
Have you tried running this without requiring 'bigdecimal'? I've run some files that have it recently without requiring it and they've worked fine. This could be a holdover from working with 2.5.3 previously.

Reply:
Looking now. Thanks @NoahZinter !

Reply:
You're right 🎉 -- I'll update. Great catch!

Reply:
Oh, but actually the ItemRepo fails the spec harness without that.

Reply:
Don't need it in the Item class tho